nntrainer/layers/split_layer.cpp

```diff
@@ -133,12 +135,14 @@ void SplitLayer::calcDerivative(RunLayerContext &context) {
     output_.reshape(output_reshape_helper);
 
     for (unsigned int batch = 0; batch < input_.batch(); batch++) {
-      Tensor dest_tensor = Tensor::Map(input_.getAddress(batch, 0, idx, 0),
-                                       input_reshape_helper.width(),
-                                       {1, 1, 1, input_reshape_helper.width()});
-      const Tensor source_tensor = Tensor::Map(
-        output_.getAddress(batch, 0, 0, 0), output_reshape_helper.width(),
-        {1, 1, 1, output_reshape_helper.width()});
+      Tensor dest_tensor =
+        Tensor::Map(input_.getAddress(batch, 0, idx, 0),
+                    input_reshape_helper.width() * sizeof(float),
```

Comment:
replace `sizeof(float)` with `sizeof(input_.getValue(0))` or `create tensor.sizeElement()`.

Reply:
we can have a tensor size type defined as a template meta function. using decltype. because sizeof(input_.getValue(0)) is runtime type inference. I'll do this as a separate PR.

Reply:
https://github.com/nnstreamer/nntrainer/issues/1515

Reply:
cool.